cmd/osbuild-worker/jobimpl-ostree-resolve.go

```diff
@@ -51,7 +52,15 @@ func (impl *OSTreeResolveJobImpl) Run(job worker.Job) error {
 	logWithId.Infof("Resolving (%d) ostree commits", len(args.Specs))
 
 	for i, s := range args.Specs {
-		reqParams := ostree.SourceSpec(s)
+		reqParams := ostree.SourceSpec{}
+		reqParams.URL = s.URL
+		reqParams.Ref = s.Ref
+		reqParams.Proxy = impl.RepositoryMTLSConfig.Proxy.String()
+		reqParams.MTLS = &ostree.MTLS{
+			CA:         impl.RepositoryMTLSConfig.CA,
+			ClientCert: impl.RepositoryMTLSConfig.MTLSClientCert,
+			ClientKey:  impl.RepositoryMTLSConfig.MTLSClientKey,
+		}
 		commitSpec, err := ostree.Resolve(reqParams)
```

Comment:
shouldn't we only add this in case the url matches, like in the depsolve job? That way it should be a smooth transition from rhsm to this (if it doesn't error out if both are set and mtls takes priority, not sure).